src/unstract/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -16,22 +17,49 @@
       "type": "string",
       "title": "URL",
       "format": "uri",
-      "default": "http://unstract-llm-whisperer:3005",
-      "description": "Provide the URL of the LLM Whisperer server."
+      "default": "https://llmwhisperer-api.unstract.com",
+      "description": "Provide the URL of the LLM Whisperer service."
     },
-    "extraction_mode": {
+    "unstract_key": {
```

Comment:
If this key needs to be entered by user, how will we be able to make it default for OSS user? User needs to manully set this up? @arun-venkataswamy @chandrasekharan-zipstack

Reply:
@gaya3-zipstack yes the user needs to manually set it up. Will add steps in the `Unstract` repo's README.md
We could have a copy of that in the adapters packages too